install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -179,11 +179,23 @@ following:
 ----
 /<example_fs> *(rw,root_squash)
 ----
-- The firewall must be configured to allow traffic to the mount point. The default port is 2049:
+- The firewall must be configured to allow traffic to the mount point, for `NFSv4` the default port is 2049(nfs).  
+  For `NFSv3` there are three ports that need to be configured, 2049(nfs), 20048(mountd) and 111(port mapper):
++
+`NFSv4:`
 +
 ----
 # iptables -I INPUT 1 -p tcp --dport 2049 -j ACCEPT
 ----
++
+`NFSv3:`
```

Comment:
@adellape  Well, thought would be nice to put some emphasis on the differentiation since I'm calling out differences between v3 and v4 to make it easier for users to easily see that.  But I'll do what ever you think is the standard way to represent this